server/community.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"time"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/mlog"
+)
+
+func (s *Server) addHacktoberfestLabel(pr *model.PullRequest) {
+	if pr.State == "closed" {
+		return
+	}
+
+	// Ignore PRs not created in october
+	if pr.CreatedAt.Month() != time.October {
+		return
+	}
+
+	isContributorOrgMember, err := s.isOrgMember("mattermost", pr.Username)
+	if err != nil {
+		mlog.Error("Error getting org membership", mlog.Err(err), mlog.Int("PR", pr.Number), mlog.String("Repo", pr.RepoName))
```

Comment:
To confirm, by core committer in PR description, you're referring to anyone not in Mattermost GitHub org?

Reply:
Yes

Reply:
Maybe GitHub org member is a better term.